Compiler/test/EAUtils.jl

```diff
@@ -351,12 +343,19 @@ Note that this version does not cache the analysis results.
 
 - `world::UInt = Base.get_world_counter()`:
   controls the world age to use when looking up methods, use current world age if not specified.
-- `interp::AbstractInterpreter = EscapeAnalyzer(world, EscapeCache())`:
+- `token_symbol::Symbol = gensym(:EA)`:
+  specifies the cache token symbol to use, by default a new symbol is generated to ensure
```

Comment:
`gensym` is somewhat expensive for this (since it is permanently allocated), and only ensures weak local uniqueness (it repeats values commonly). Is this the only way we have of being able to disable the cache?

Reply:
Yes, allocating a `mutable struct` might be a better approach. Since EAUtils.jl is primarily used for debugging and testing purposes, the impact on runtime performance from indiscriminately creating code caches should be exclusive.

Reply:
Apologies, it seems your concern was indeed valid. `@benchmark code_escapes(...)` is used frequently, and in such cases, it could lead to undesirable situations. JuliaLang/julia#56868